lib/pbench/server/api/resources/query_apis/elasticsearch_api.py

```diff
@@ -12,12 +20,13 @@ def __init__(self, config: PbenchServerConfig, logger: Logger):
         """
         __init__ Configure the Elasticsearch passthrough class
 
-        NOTE: there are three "expected" JSON input parameters, but only two of
-        these are required and therefore appear in the schema.
-
-        TODO: the schema could be extended to include both the type and a
-        "required" flag; is that worthwhile? This would allow automatic type
-        check/conversion of optional parameters, which would be cleaner.
+        NOTE: This API is obsolescent. We've talked about retaining it as a
+        potentially "privileged" mechanism to gain direct access to the
+        Elasticsearch backend, but this isn't defined. We should likely
```

Comment:
We could limit it to admin use, for the moment, if that's not too hard...or, yeah, we should disable it in some fashion lest we forget.  (We _could_ just make `assemble()` unconditionally report `Forbidden`....)

Reply:
I don't want to get into "fixing" it for this PR, but I created [PBENCH-717](https://issues.redhat.com/browse/PBENCH-717) to track cleanup on this aisle.